### PR TITLE
feat: add fish --no-execute diagnostics

### PR DESCRIPTION
**Pull request by timbedard**
`fish --no-execute $FILENAME` will give basic diagnostics for `.fish` files.

### CODE REVIEW

File: lua/null-ls/builtins/diagnostics/fish.lua [hunk 1]
```diff
@@ -0,0 +1,25 @@
+local h = require("null-ls.helpers")
+local methods = require("null-ls.methods")
+
+return h.make_builtin({
+    name = "fish",
+    meta = {
+        url = "https://github.com/fish-shell/fish-shell",
+        description = "Code audit tool for Python.",
```
Comment (jose-elias-alvarez) [inline lua/null-ls/builtins/diagnostics/fish.lua:8]: I think this is probably left over?

**Review by jose-elias-alvarez — COMMENTED** [review]
Other than the description, looks good!

**Comment by timbedard** [comment]
Whoops. Updated.

**Comment by jose-elias-alvarez** [comment]
LGTM, thanks!

**Comment by timbedard** [comment]
Happy to contribute!

**Pull request merged**